File: trunk/project/ponto/source/sistemas/admin/models/Associado.php
<?php
/**
 * Modelo da tabela associado
 *
 * @name Associado
 * @author Alexis Moura
 * @package _models
 * @version 1.0
 */
class Associado extends Axs_Db_Table
{
    /**
     * Array de dados do modelo
     *
     * @name $_data
     * @access public
     * @var array
     */
    public $_data = array(
                   	                 'idMembro' => "",
                   	                 'idAssociacao' => "",
                   	                 'idAssociado' => "",
                                  );
    /**
     * Array de validação do modelo
     *
     * @name $_validators
     * @access protected
     * @var array
     */
    protected $_validators = array(
                   	                 'idMembro' => array( 'notEmpty','presence'=>'required'),
                   	                 'idAssociacao' => array( 'notEmpty','presence'=>'required'),
                   	                 'idAssociado' => array( 'notEmpty','presence'=>'required'),
                                  );
    /**
     * Array de descrição dos dados
     *
     * @name $descriptionFields
     * @access public
     * @var array
     */
    public $descriptionFields = array(
                   	                 'idMembro' => 'Membro',
                   	                 'idAssociacao' => 'Associado',
                   	                 'idAssociado' => 'Código',
                                  );
					

}

?>

File: trunk/project/ponto/PontoEncontro.Domain/Entidades/Associado.cs
using System;
using System.Collections.Generic;

namespace PontoEncontro.Domain
{
    [Serializable]
    public class Associado
    {


        public Associado()
        {
        }

        public Associado(int Idassociado, Membro Idmembro, Associacao Idassociacao)
        {
            this.Idassociado = Idassociado;
            this.Idmembro = Idmembro;
            this.Idassociacao = Idassociacao;
        }

        public virtual int Idassociado { get; set; }
        public virtual Membro Idmembro { get; set; }
        public virtual Associacao Idassociacao { get; set; }

    }
}
